src/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+  padding: 0;
+  --height: 300px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  min-width: var(--height);
+  line-height: var(--height);
+  text-align: center;
+  flex-grow: 1;
+  flex-basis: 33.3%;
+}
+
+.block1 {
+  background-color: red;
+}
+
+.block2 {
+  background-color: #c00 ;
+}
+
+.block3 {
+  background-color: #900;
+}
+
+.block4 {
+  background-color: #600;
+  order: 3;
+}
+
+.block5 {
+  background-color: #300;
+  order: 2;
+}
+
+.block6 {
+  background-color: black;
+}
+
+@media (max-width: 899px) {
```

Comment:
use min-width instead, it'd be easier

Reply:
ok, done
{
but it wasn`t to  hard even with 899.9999))
}